bob.spec

```diff
@@ -20,58 +18,61 @@ Requires: make-gnu >= 4.2-2
 Requires: python3 >= 3.6
 
 Source0: https://github.com/IBM/ibmi-bob/archive/refs/tags/v%{version}.tar.gz
-Source1: https://github.com/BrianGarland/CRTFRMSTMF/archive/master.tar.gz
 
 %description
-Better Object Builder, or Bob, is a free and open source build system for the IBM i platform that is used to build native "QSYS" objects.
+Better Object Builder, or Bob, is a free and open source build system for the
+IBM i platform that is used to build native "QSYS" objects.
 Here's what makes Bob different.
-- Speed. Bob only compiles objects that need recompiling, like from new or changed source code.
-- Reliability. Bob understands the relationships between your objects, so if an item changes, then it and everything depending on it will be rebuilt.
-- Industry standard. Object dependencies are specified using standard makefile syntax, and the actual build engine is GNU Make -- exactly like tens of thousands of Linux and Unix software projects.
-- Flexibility. Most objects defined to Bob typically build using your default values. Have a program that requires a custom activation group or a data area that needs to be created with a certain value? No problem, overriding compile parameters is trivial, and writing custom recipes for special objects is very straightforward. If you can code it, you can build it.
-- Ease of use. Invoking a build of an entire codebase is done with just a single command. Or, if the Rational Developer for i integration pieces are installed, a single button click.
+- Speed. Bob only compiles objects that need recompiling, like from new or 
+  changed source code.
+- Reliability. Bob understands the relationships between your objects, so if an
+  item changes, then it and everything depending on it will be rebuilt.
+- Industry standard. Object dependencies are specified using standard makefile
+  syntax, and the actual build engine is GNU Make -- exactly like tens of 
+  thousands of Linux and Unix software projects.
+- Flexibility. Most objects defined to Bob typically build using your default
+  values. Have a program that requires a custom activation group or a data area
+  that needs to be created with a certain value? No problem, overriding compile
+  parameters is trivial, and writing custom recipes for special objects is very
+  straightforward. If you can code it, you can build it.
+- Ease of use. Invoking a build of an entire codebase is done with just a 
+  single command. Or, if the Rational Developer for i integration pieces are
```

Comment:
Remove the Rational Developer for i sentence for now, since we don't have integration right now